hail_search/__main__.py

```diff
@@ -0,0 +1,16 @@
+from aiohttp import web
+
+from hail_search.web_app import init_web_app
+
+
+def run():
+    app = init_web_app()
+    web.run_app(
+        app,
+        host='0.0.0.0',  # nosec
```

Comment:
It looks like a backend service web app without authentication support. It should not be accessible from external.

Reply:
Correct. The hail backend service will only be accessible via an internal, unauthenticated kubernetes service, so that only seqr can access it. Actually setting up the deployment is out of scope of this PR, but you can see the kubernetes yaml that I used to get this working in a test environment here: https://github.com/broadinstitute/seqr/blob/62084009dd60a35eef1c85608ad0c8a3e2a90cf8/deploy/kubectl_helpers/utils/hail-search.yaml